docs/core/diagnostics/eventpipe.md

```diff
@@ -0,0 +1,82 @@
+---
+title: EventPipe Overview
+description: Learn about EventPipe and how to use it for tracing your .NET applications to diagnose performance issues.
+ms.date: 11/09/2020
+ms.topic: overview
+---
+
+# EventPipe
+
+EventPipe is a runtime (CoreCLR) component that can be used as one of tracing egression points, similar to ETW or LTTng. The goal of EventPipe is to allow .NET developers to easily trace their .NET applications without having to rely on OS-native components such as ETW or LTTng.
```

Comment:
I don't want to call out Mono yet before we ship .NET 6.

Reply:
I was thinking of whether we should remove the callout entirely rather than including Mono.

Reply:
Hmm.... I'm not so sure. I guess the doc itself is in Core so I guess it'd be fine to remove it? The goal was to disambiguate CoreCLR with Mono because both are now '.NET' - but I'll remove it for now. Thanks!